.gitmodules

```diff
@@ -13,3 +13,12 @@
 [submodule "bolt-contracts/lib/eigenlayer-contracts"]
 	path = bolt-contracts/lib/eigenlayer-contracts
 	url = https://github.com/layr-labs/eigenlayer-contracts
+[submodule "lib/openzeppelin-contracts-upgradeable"]
+	path = lib/openzeppelin-contracts-upgradeable
+	url = https://github.com/OpenZeppelin/openzeppelin-contracts-upgradeable
+[submodule "bolt-contracts/lib/openzeppelin-contracts-upgradeable"]
+	path = bolt-contracts/lib/openzeppelin-contracts-upgradeable
+	url = https://github.com/OpenZeppelin/openzeppelin-contracts-upgradeable
```

Comment:
this seems like a duplicate module, can you check which one we're actually using?